Training/Training/change.c

```diff
@@ -0,0 +1,70 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// change.c
+// Program to perform and test arithmetic operations on complex numbers.
+// ------------------------------------------------------------------------------------------------
+
+#include<stdio.h>
+#include<errno.h>
+#include<stdbool.h>
+#include<stdlib.h>
+#include<string.h> // USED
+
+void OptimalChangeCalculator (int* coins, int rem, int* denominations) {
+   for (int i = 0; i < 4; i++) {
+      coins[i] = rem / denominations[i];
+      rem %= denominations[i];
+   }
+}
+
+static void Testcases (int* denominations) {
+   int input[] = { 95,75,67,79 }, expected[][4] = { 9,1,0,0,7,1,0,0,6,1,1,0,7,1,2,0 }, output[4];
+   printf ("\nTestcases\n~~~~~~~~~\n");
+   for (int i = 0; i < sizeof (input) / sizeof (input[0]); i++) {
+      OptimalChangeCalculator (output, input[i], denominations);
+      int j;
+      for (j = 0; j < 4; j++)
+         if (expected[i][j] != output[j]) {
+            printf ("Case %d: FAILED\n", i + 1);
+            break;
```

Comment:
Instead of breaking you can continue; Check for j is not required.

Reply:
If more than 1 expected value fails, the statement will get printed as many times.